src/Altinn.Apps/AppFrontend/react/shared/src/utils/language.ts

```diff
@@ -65,12 +65,12 @@ const removeStyling = (node: any): React.ReactElement | void | null => {
   }
 };
 
-const replaceParameters = (nameString: any, params: any[]) => {
+const replaceParameters = (nameString: string, params: string[]) => {
   let index = 0;
   // eslint-disable-next-line no-restricted-syntax
   for (const param of params) {
```

Comment:
Why not:
```JavaScript
let mutatingString = nameString.slice()
params.forEach((param, index) => {
  mutatingString = mutatingString.replaceAll(`{${index}}`, param);
}
return mutatingString;
```
If you are mutating the string anyway, why return anything?

Reply:
Agree with the refactoring, disagree with the return. Strings are immutable, we cant mutate the value direclty? We have to return the new value for this to work. Or what are you proposing?